rosidl_typesupport_opensplice_cpp/bin/rosidl_typesupport_opensplice_cpp

```diff
@@ -59,9 +60,18 @@ def main(argv=sys.argv[1:]):
         return 1
     if rc:
         return rc
+    dds_interface_files = []
+    if not os.path.exists(args.dds_interface_files_file):
+        print("File does not exist: {0}".format(args.dds_interface_files_file), file=sys.stderr)
+        return 1
+    with open(args.dds_interface_files_file, 'r') as f:
+        for line in f.readlines():
+            if not line.strip():
+                continue
+            dds_interface_files.append(line.strip())
```

Comment:
I wouldn't expect there to be one, but I think this is more defensive to check. For example, in case someone was hand crafting a file for testing and left trailing whitespace or an extra line at the end of the file. I'll make it more concise, but I'll leave it defensive.